Analysis/Tasks/jetfinder.cxx

```diff
@@ -96,7 +94,7 @@ struct JetFinderTask {
 
     std::vector<fastjet::PseudoJet> inputParticles;
     for (auto& track : fullTracks) {
-      auto energy = std::sqrt(track.p2() + mPionSquared);
+      auto energy = std::sqrt(track.p() * track.p() + mPionSquared);
```

Comment:
I assume p() has a sqrt inside? does it really make sense to have it rather than just p2?

Reply:
I've tried to keep data model as it was before. We can redefine what we want calculated after people have "played" with new capabilities.

Reply:
ok, but here the old version was using p2(), why changing it then? Anyways, fine for me.

Reply:
'p2' was from the implementation with no functions. So far it was used only in this file, it seems.